src/main/java/bio/terra/cli/command/config/set/AppLaunch.java

```diff
@@ -10,7 +10,7 @@
 @Command(name = "app-launch", description = "Configure the ways apps are launched.")
 public class AppLaunch extends BaseCommand {
 
-  @CommandLine.Parameters(index = "0", description = "App launch mode: ${COMPLETION-CANDIDATES}")
+  @CommandLine.Parameters(index = "0", description = "App launch mode: ${COMPLETION-CANDIDATES}.")
```

Comment:
Should this description start with a verb?

Reply:
Parameter and option descriptions don't need to start with a verb, just the description for commands or command groups.